### DISCUSSION TITLE
Getting models and some other values

### DISCUSSION THREAD

**Discussion by SteelAlloy** [SeaQL/sea-orm#1797] [Q&A]
Hi, 
I've been trying to rewrite my code from diesel, but I can't seem to solve this particular problem.

With diesel, I could define some struct :
```rust
pub struct CakeData {
    cake: Cake,
    some_other_property: i64,
}
```

and then load my data inside it :
```rust
let res : Vec<CakeData> = query.load(&mut ctx.db_conn())
```

---

With SeaORM I can't find any way to do it this way, it seems like I need to write a struct with every cake column + my custom properties
```rust
#[derive(Copy, Clone, Debug, EnumIter, DeriveColumn)]
enum QueryAs {
    CakeName,
    // ... need to write again every column ?
    NumOfCakes,
}

let res: Vec<(Cake, i64)> = cake::Entity::find()
    .column_as(Expr::cust("COUNT(*) OVER ()"), "remaining_count")
    .into_values::<_, QueryAs>()
    .all(&db)
    .await?;
```

`Vec<(Cake, i64)` would also be ok !

Is there a way to achieve this result? thanks

**Comment by tyt2y3** [comment]
I think you can try https://www.sea-ql.org/SeaORM/docs/advanced-query/custom-select/#select-partial-model

**Reply by SteelAlloy** [reply]
Thanks, although that's specifically what I want to avoid. My tables have a lot of columns (32+) and it would be tedious to rewrite them all each time.
Maybe this https://github.com/SeaQL/sea-orm/issues/739 is what I'm looking for?

**Reply by tyt2y3** [reply]
May be, we can have an additional macro attribute, `#[sea_orm(skip)]`, similar to `#[sea_orm(ignore)]`, but would still be included in `FromQueryResult`.
Such that you can add the extra field into the Model.
